src/ompl/util/RandomNumbers.h

```diff
@@ -43,8 +43,18 @@
 #include <boost/random/variate_generator.hpp>
 #include <cassert>
 
+#include <ompl/util/ClassForward.h>
+
+//The basic Eigen include
+#include <Eigen/Core>
```

Comment:
Maybe use the PIMPL pattern in the ProlateHyperSperoid class and hide the Eigen includes in the cpp file (to speed up compilation of everything that includes RandomNumbers.h).